src/actions/onboarding.ts

```
@@ -197,3 +380,142 @@ export async function submitOnboarding(formData: FormData) {
+  } catch (error) {
+    console.error("Error finalizing startup onboarding:", error);
+    throw new Error(
+      error instanceof Error
+        ? error.message
+        : "Failed to finalize startup onboarding"
+    );
+  }
+
+  // Revalidate and redirect outside of try-catch to avoid catching redirect errors
+  revalidatePath("/home");
+  redirect("/home");
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will return from the function, ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)